src/components/LoginForm.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+
+export default class LoginForm extends Component {
+  constructor(props) {
+    super(props);
+    this.handleSubmit = this.handleSubmit.bind(this)
+  }
+
+    handleSubmit(event){
+        event.preventDefault()
+        const loginData = {}
+        loginData.username = event.target.username.value
+        loginData.password = event.target.password.value
```

Comment:
In the future you probably need to add some validation, so you have to do those inputs are controlled, if not its okay